### DISCUSSION TITLE
[QUESTION] longhorn-manager README is outdated

### DISCUSSION THREAD

**Discussion by hookak** [longhorn/longhorn#8416] [General]
## Question

Hello, I'm new to longhorn.

I try to build [longhorn-manager](https://github.com/longhorn/longhorn-manager) and deploy it.
The problem is, the README for that project is outdated.
I manually modified the longhorn-manager image in the [deploy/longhorn.yaml](https://github.com/longhorn/longhorn/blob/master/deploy/longhorn.yaml) and deployed it.
The guide is a bit confusing.
Do you have any plans to update the guide for newcomers?

## Environment

 - Longhorn version: longhorn-manager master branch

**Comment by mantissahz** [comment]
Yeah, thank you. I will handle the README.md with this ticket https://github.com/longhorn/longhorn/issues/6914.
Could you point out which parts are outdated or confusing you? Thanks.

**Reply by hookak** [reply]
I did not see issue https://github.com/longhorn/longhorn/issues/6914. This is exactly the same comment as that issue.

And,it would also be nice to have a way to upgrade longhorn-manager to the latest image built.
We need something similar to the https://github.com/longhorn/longhorn/blob/master/dev/scripts/lm-update.sh script

**Comment by hookak** [comment]
Closing this discussion because https://github.com/longhorn/longhorn/issues/6914 has been merged
